### DISCUSSION TITLE
Issue with Socket

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#12] [Discuss about technical stack] [answered]
Hello, truestar.
Fix socket error that you made.
This is critical error.

**Comment by Daniel-Kennison** [comment]
I will fix

**Comment by creative2113** [comment, marked as answer]
I am a full stack engineer.
I will fix in a short time.

**Reply by noahstoner** [reply]
I am expecting your activity

**Reply by creative2113** [reply]
what can I do for that.
